queue/message.go

```diff
@@ -102,18 +104,18 @@ func (mh *messageHandler) Handle(message sqs.Message) (*runbook.ActionResultPayl
 	return result, nil
 }
 
-func (mh *messageHandler) execute(mappedAction *conf.MappedAction, messageBody string) (string, error) {
+func (mh *messageHandler) execute(mappedAction *conf.MappedAction, message sqs.Message) (string, string, error) {
```

Comment:
you can use a pointer for sqs message not to copy the value. also it is an already private method